app/src/main/java/com/duckduckgo/app/browser/autocomplete/BrowserAutoCompleteSuggestionsAdapter.kt

```diff
@@ -59,22 +65,21 @@ class BrowserAutoCompleteSuggestionsAdapter(
     }
 
     override fun getItemCount(): Int {
-        if (suggestions.isNotEmpty()) {
-            return suggestions.size
+        if (suggestions.isEmpty() && phrase.isNotBlank()) {
```

Comment:
This fixes a bug across the our app where "No suggestions" would flash up on the screen.